website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -18,32 +18,29 @@ Policy sets are collections of policies you can apply globally or to specific [p
+## Comparing policy checks and evaluations
 
-Though policy checks and policy evaluations seem like synonyms, there is a fundamental difference between the policy check and policy evaluation workflow.
+Sentinel policy sets can run as both policy checks and evaluations, while OPA policy sets only run as evaluations. Policy checks and evaluations can access different types of data and enable slightly different workflows.
 
-### Policy Checks
+### Policy checks
 
-Sentinel policies are currently run as a policy check. This is the default workflow for Sentinel.
-Policy checks use the latest version of the Sentinel runtime and have access to cost estimation data.
+Only Sentinel policies can run as policy checks, and it is the default mode for Sentinel policy sets. Checks can access cost estimation data but can only use the latest version of Sentinel.
 
-### Policy Evaluations
+### Policy evaluations
 
-Policy evaluations are run within the [Terraform Cloud Agent](/terraform/cloud-docs/agents) in Terraform Cloud's infrastructure.
-Policy evaluations do not have access to cost estimation data.
+OPA policy sets can only run as policy evaluations, and you can enable policy evaluations for Sentinel policy sets by selecting the `Enhanced` policy set type. Policy evaluations run within the [Terraform Cloud Agent](/terraform/cloud-docs/agents) in Terraform Cloud's infrastructure. 
 
 -> **Note:** Sentinel policy evaluations are currently in beta.
-Sentinel policies can run as a policy evaluation on an opt-in basis.
 
-## Management Workflows
+For Sentinel policy sets, using policy evaluations lets you:
+- Enable overrides for soft-mandatory and hard-mandatory policies, letting any user with [Manage Policy Overrides permission](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policy-overrides) proceed with a run in the event of policy failure.
+- Select a specific Sentinel runtime version for the policy set.
 
-You can use the following workflows to manage policies and policy sets for your Terraform Cloud organization:
-- **Individually Managed:** Add policies directly in the Terraform Cloud UI, and Terraform Cloud stores your policy code. This workflow is ideal for initial experimentation with policy enforcement, but we do not recommend it for organizations with large numbers of policies.
-- **Version Control:**  Connect Terraform Cloud to a version control repository containing a policy set. When you push changes to the repository, Terraform Cloud automatically uses the updated policy set.
-- **Automated:** Push versions of policy sets to Terraform Cloud with the [Terraform Cloud Policy Sets API](/terraform/cloud-docs/api-docs/policy-sets#create-a-policy-set-version). For Sentinel only, you can also use the  `tfe` provider [`tfe_policy_set`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/policy_set) resource. This workflow is ideal for automated Continuous Integration and Deployment (CI/CD) pipelines.
```

Comment:
> For Sentinel only, you can also use the  `tfe` provider [`tfe_policy_set`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/policy_set) resource.

@mrinalirao I just want to double check the above with you as I was under the impression that we support both Sentinel and OPA.

Reply:
yes, the docs seem to be incorrect, we support both Sentinel and OPA for tfe_policy_set provider